charts/openshift-metering/templates/hive/hive-s3-compatible-credentials-secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.hive.spec.config.s3Compatible.createSecret -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hive-s3-compatible-credentials
+data:
+{{- if .Values.hive.spec.config.s3Compatible.accessKeyID }}
+  s3-compatible-access-key-id: {{ .Values.hive.spec.config.s3Compatible.accessKeyID | b64enc | quote}}
```

Comment:
Let's use the same keys in the secret as s3, not something new. Eg `aws-access-key-id` and `aws-secret-access-key`